product_supplierinfo_for_customer_sale/tests/test_product_name_search.py

```diff
@@ -70,7 +70,7 @@ def test_10_find_product_customer_code(self):
         self.assertFalse(self.product_1.customer_ids)
 
         # Search by product customer code
-        product_names = self.product.name_search(name="code_test")
+        product_names = self.product.name_search(name="code_test", operator="ilike")
```

Comment:
I think this change is not required, `ilike` is [still the default operator](https://github.com/odoo/odoo/blob/e2f9c21f06837cb7bfdf2547a05633e563164d09/odoo/models.py#L1581) in 16.0

Same for the below similar cases